src/session.ts

```diff
@@ -90,6 +93,13 @@ export class DebugSession implements IDebugger.ISession {
     });
   }
 
+  /**
+   * Request code execution.
+   */
+  async execute(code: string): Promise<void> {
+    void this._sendExecuteMessage({ code });
```

Comment:
Should this be `await` instead of `void`?

Reply:
Right, good catch!